api/tests/unit-tests/schemas/test_metrics.py

```diff
@@ -382,3 +382,127 @@ def test_mIOUMetric():
             for key in iou_metric.db_mapping(evaluation_id=1)
         ]
     )
+
+
```

Comment:
Need tests for AnswerRelevance and Coherence

Reply:
has this been addressed?